bjoern/py2py3.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _PY2PY3_H
+#define _PY2PY3_H
+
+#if PY_MAJOR_VERSION >= 3
+#define _Bytes_AS_DATA(obj) PyBytes_AS_STRING(obj)
+#define _Unicode_AS_DATA(obj) PyUnicode_AsUTF8(obj)
+#define _Bytes_FromString(name) PyBytes_FromString(name)
+#define _Unicode_FromString(name) PyUnicode_FromString(name)
+#define _Bytes_FromStringAndSize(data, len) PyBytes_FromStringAndSize(data, len)
+#define _Unicode_FromStringAndSize(data, len) PyUnicode_FromStringAndSize(data, len)
+#define _Bytes_GET_SIZE(obj) PyBytes_GET_SIZE(obj)
+#define _Unicode_GET_SIZE(obj) PyUnicode_GET_LENGTH(obj)
+#define _Bytes_Check(obj) PyBytes_Check(obj)
+#define _Unicode_Check(obj) PyUnicode_Check(obj)
+#define _Bytes_Resize(obj, len) _PyBytes_Resize(obj, len);
+#define _Unicode_Resize(obj, len) PyUnicode_Resize(obj, len);
+#define _FromLong(n) PyLong_FromLong(n)
+#define _File_Check(file) 1 /* To Be Fixed */
```

Comment:
New changes pushed. Now using hasattr() + callable() to check for fileno exists and is callable.